TROUBLESHOOTING.md

```diff
@@ -99,12 +99,17 @@ If you think you're hitting an out-of-memory situation, you can alleviate this p
 
 Pylance (by default) runs using VS Code's Node.js executable (which has the 4GB limit). 
 
-To specify your own Node.js executable, set this setting in your User settings.json and restart VS Code:
+To specify your own Node.js executable, set this setting in your user settings.json and restart VS Code:
 
 ```json
 "python.analysis.nodeExecutable": "<path to node.js exe>"
 ```
 
+Your user settings.json depends upon how you're connecting:
+
+- Local - Should by your User settings.json
```

Comment:
Is this clear? Seems to be saying, "Your user settings.json...is your User settings.json"

Reply:
Sorry I updated all of these to something different. Must have been after you started looking.

Reply:
Please look again.